generic_server.cc

```diff
@@ -228,7 +237,17 @@ future<> server::do_accepts(int which, bool keepalive, socket_address server_add
             auto addr = std::move(cs_sa.remote_address);
             fd.set_nodelay(true);
             fd.set_keepalive(keepalive);
-            auto conn = make_connection(server_addr, std::move(fd), std::move(addr));
+            auto conn = make_connection(server_addr, std::move(fd), std::move(addr),
+                    seastar::gate::holder(_gate_uninitialized_conns));
+
+            if (size_t c =_gate_uninitialized_conns.get_count();
```

Comment:
Does this gate serve any further purpose, than counting the connections?

Reply:
no